api/swagger.yaml

```diff
@@ -6229,6 +6229,10 @@ paths:
           in: "query"
           description: "If “1”, “true”, or “True” then it will be an error if unpacking the given content would cause an existing directory to be replaced with a non-directory and vice versa."
           type: "string"
+        - name: "copyUIDGID"
+          in: "query"
+          description: "If “1”, “true”, or “True” then it will copy UID/GID maps to the dest file"
```

Comment:
I see this is documented like this on other places as well, but perhaps we should just document `1` and `true`. 

Strictly speaking, any non-empty value should work; https://github.com/moby/moby/blob/4f0d95fa6ee7f865597c03b9e63702cdcb0f7067/api/server/httputils/form.go#L9-L13

But I don't think we should officially support all of that (so officially support just `1` and `true`)